client/src/common/Modal.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <transition name="modal-fade">
+    <ModalWrapper>
+      <ModalContent>
+        <header>
+          <ModalCloseBtn @click="close">x</ModalCloseBtn>
+          <slot name="header">Default title</slot>
+        </header>
+        <section>
+          <slot></slot>
+        </section>
+        <footer>
+          <slot name="footer"></slot>
+        </footer>
+      </ModalContent>
+    </ModalWrapper>
+  </transition>
+</template>
+
+<style>
+.modal-fade-enter {
```

Comment:
We definitely should decide what kind of CSS and how we gonna write it.
I thought we should you styled component for everything.
I don't want you to change something here, but I will raise this point on retro :)

Reply:
I guess it is kind of global class created by Vue for transition. If you have any suggestions how to handle it with styled components it will be great. Maybe here is the solution but actually I didn't find it